percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -0,0 +1,783 @@
+#!/bin/sh
+
+shell_quote_string() {
+  echo "$1" | sed -e 's,\([^a-zA-Z0-9/_.=-]\),\\\1,g'
+}
+
+usage () {
+    cat <<EOF
+Usage: $0 [OPTIONS]
+    The following options may be given :
+        --builddir=DIR      Absolute path to the dir where all actions will be performed
+        --get_sources       Source will be downloaded from github
+        --build_src_rpm     If it is set - src rpm will be built
+        --build_source_deb  If it is set - source deb package will be built
+        --build_rpm         If it is set - rpm will be built
+        --build_deb         If it is set - deb will be built
+        --build_tarball     If it is set - tarball will be built
+        --install_deps      Install build dependencies(root privilages are required)
+        --branch            Branch for build
+        --repo              Repo for build
+        --psm_ver           PSM_VER(mandatory)
+        --psm_release       PSM_RELEASE(mandatory)
+        --mongo_tools_tag   MONGO_TOOLS_TAG(mandatory)
+        --ftindex_tag       FTINDEX_TAG(mandatory)
+        --jemalloc_tag      JEMALLOC_TAG(mandatory)
+        --rocksdb_tag       ROCKSDB_TAG(mandatory)
+        --tokubackup_branch TOKUBACKUP_BRANCH(mandatory)
+        --debug             build debug tarball
+        
+        --help) usage ;;
+Example $0 --builddir=/tmp/PSMDB --get_sources=1 --build_src_rpm=1 --build_rpm=1
```

Comment:
Why example does not contain all those mandatory arguments like ftindex_tag, rocksdb_tag etc